libs/pages/application/feature/src/lib/application-page.tsx

```diff
@@ -22,7 +22,7 @@ export function ApplicationPage() {
   useEffect(() => {
     setTimeout(() => {
```

Comment:
So, why you didn't remove the setTimeout?

Reply:
Yep, we can remove that, it's a forget